lib/log.js

```diff
@@ -178,25 +178,59 @@ Lug.prototype.activityEvent = function (event, request, data) {
     return P.resolve()
   }
 
+  // log a flowEvent for all activityEvents
+  this.flowEvent(event, request)
+
+  var info = {
+    event: event
+  }
+
+  if (request.headers['user-agent']) {
+    info.userAgent = request.headers['user-agent']
+  }
+
+  optionallySetService(info, request)
+
+  Object.keys(data).forEach(function (key) {
+    info[key] = data[key]
+  })
+
+  this.logger.info('activityEvent', info)
+  this.statsd.write(info)
+
+  return P.resolve()
```

Comment:
> Instead of an empty promise, should we return the promise returned by this.flowEvent?

This one feels weird, returning a promise from a flowEvent when we called activityEvent